mozilla_django_oidc/utils.py

```diff
@@ -39,3 +39,19 @@ def is_authenticated(user):
     if _less_than_django_1_10:
         return user.is_authenticated()
     return user.is_authenticated
+
+
+def get_oidc_scopes():
```

Comment:
Well i don't see the value of having a list of strings and I prefer the less complicated approach.